Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionGenericCEF.yaml

```diff
@@ -0,0 +1,302 @@
+Parser:
+    Title:  Network Session ASIM parser for Generic CEF
+    Version: '0.1'
+    LastUpdated: Feb 17, 2022
+Product:
+    Name: Common Event Format
+Normalization:
+    Schema: NetworkSession
+    Version: '0.2.2'
+References:
+    - Title: ASIM Network Session Schema
+      Link: https://aka.ms/AzSentinelNetworkSessionDoc
+    - Title: ASIM
+      Link: https://aka.ms/AzSentinelNormalization
+    - Title: Micro Focus Security ArcSight Common Event Format [pdf]
+      Link: https://www.microfocus.com/documentation/arcsight/arcsight-smartconnectors/pdfdoc/common-event-format-v25/common-event-format-v25.pdf
+    - Title: CEF and CommonSecurityLog field mapping
+      Link: https://docs.microsoft.com/en-us/azure/sentinel/cef-name-mapping
+Description: |
+    This ASIM parser supports normalizing Common Event Format to the ASIM Network Session schema.
+ParserName: ASimNetworkSessionGenericCEF
+EquivalentBuiltInParser: _ASim_NetworkSession_GenericCEF
+ParserParams:
+    - Name: disabled
+      Type: bool
+      Default: false
+ParserQuery: |
+    let ASimNwCSLParsers = datatable(DeviceVendor:string, DeviceProduct:string)[
+      "Palo Alto Networks", "PAN-OS"
+      , "Zscaler" , "NSSFWlog"];
+    let ALLOW_TERMS = dynamic(['allow', 'allowed', 'permit', 'accept', 'start']);
+    let RESET_TERMS = dynamic(['reset', 'rst']);
```

Comment:
First, you don't use those....

Also, No Deny terms? Well, actually you have them in the list below. Also for other actions.